src/router/resolvers/index.ts

```diff
@@ -558,6 +558,40 @@ export const DowSummaryPathResolver = (current: string, direction: string): stri
   return OfferingDetailsPathResolver(current, direction);
 };
 
+export const IGCECannotProceedResolver = (current: string): string => {
+  const hasLegitPeriods =  Periods.periods && Periods.periods.length > 0;
+  const isCompleteDOW = DescriptionOfWork.isIncomplete === false;
+  const validToProceed = hasLegitPeriods && isCompleteDOW;
+ 
+  if (validToProceed){
+    if (current ===  routeNames.CreatePriceEstimate){
+      return routeNames.GatherPriceEstimates;
+    } else if (current == routeNames.GatherPriceEstimates) {
+      return routeNames.CreatePriceEstimate;
+    }
+  }
+  return routeNames.CannotProceed;
+};
+
+export const IGCEGatherPriceEstimatesResolver = (current: string): string => {
+  const hasLegitPeriods =  Periods.periods && Periods.periods.length > 0;
+  const isCompleteDOW = DescriptionOfWork.isIncomplete === false;
+  const validToProceed = hasLegitPeriods && isCompleteDOW;
+

```

Comment:
need to add this for when using Back button - if it's on Travel Estimates and clicking back, it's skipping the Gather Price Estimates page and going straight to the first page
<img width="639" alt="Screen Shot 2022-10-05 at 2 45 58 PM" src="https://user-images.githubusercontent.com/90333349/194144821-5df7d424-9c5c-448f-aaf5-2c1c2c84e4a1.png">

Reply:
Done, thx,.